md2html.sh

```diff
@@ -29,9 +36,17 @@ for file in "$@"; do
   fi
   # With kramdown, convert directly to HTML
   if [ "$md2html_tool" == kramdown ]; then
+    if [ "$add_toc" -ne 0 ]; then
+      sed -i '1s/^/* toc\n{:toc}\n/' "${file}"
+    fi
     # Set locale to UTF-8 because by default it is set to US-ASCII
-    LC_ALL=C.UTF-8 $md2html_tool --extension parser-gfm --input GFM \
-    --output html "${file}" > "${file%.md}.html"
+    LC_ALL=C.UTF-8 $md2html_tool --extension parser-gfm \
+      --input GFM \
+      "$toc_arg" \
```

Comment:
```suggestion
      $toc_arg \
```
otherwise you will be passing an argument that is an empty string to md2html
if shellcheck doesn't like it, add a suppression